docs/examples/infinite-scroll.md

```diff
@@ -7,10 +7,8 @@ In this example, we use a [third party library](https://github.com/Akryum/vue-ob
 
 For this to work we do 2 things:
 
-1. We tell the Results component to `stack` results
-1. We introduce a "dummy" div at the bottom that will increase the page number when it becomes visible
-
-**Info: because the example below is iframed, loading of results is a bit slowish. We recommend you take a [look here if you want optimal performances](https://s.codepen.io/rayrutjes/debug/dWBMje/XxMVwDjEywor).**
```

Comment:
It worked perfectly on my machine, I thought this was a leftover somehow. Is it slow somewhere?